models/h2o_dai_regression_auto_insurance/README.md

```diff
@@ -0,0 +1,104 @@
+# H2O Driverless AI (DAI) Scoring Pipeline Example
+
+**Dataset Used** - This example uses the Auto Insurance example originally sourced from from
+[Emcien](https://www.sixtusdakurah.com/resources/The_Application_of_Regularization_in_Modelling_Insurance_Claims.pdf).
+
+
+The prepared dataset file used in this example is available in the
+[notebooks/datasets folder](../../notebooks/datasets/auto_insurance_claims_dataset.csv)
+
+
+# Train and Download the Scoring Pipeline
+
+1. Clone this repository.
+
+2. Upload the dataset in this repository from [../../notebooks/datasets/auto_insurance_claims_dataset.csv](../../notebooks/datasets/auto_insurance_claims_dataset.csv) to
+H2O's Driverless AI and use it to auto-learn an ML model.
+
+3. Select 'Click for Actions' and then 'Predict'.
+
+4. Set the target column to 'outcome' and launch the experiment.
+
+5. After completion of the Driverless AI Experiment, download the scoring pipeline that you want to scan with Certifai.
+
+  Download the [Mojo Scoring Pipeline](https://s3.amazonaws.com/artifacts.h2o.ai/releases/ai/h2o/dai/rel-1.8.5-64/docs/userguide/scoring-mojo-scoring-pipeline.html#mojo-scoring-pipeline-files).
+
+# Setup the Certifai Scanning Environment
+
+1. Follow the [Toolkit Setup](https://cognitivescale.github.io/cortex-certifai/docs/about) in the Certifai documentation to install the Certifai toolkit into a conda
```

Comment:
The link provided does not point specifically to the Toolkit setup. Is this intentional?

Reply:
Yes (reducing future broken links), but I tweaked to at least give the path in the navigation